### ISSUE TITLE
Ability to auto-retry PRs

### ISSUE THREAD

**Issue by silasary** [bors-ng/bors-ng#414]
Sometimes, we open a PR knowing that it's not going to pass checks at the time - this might be waiting on something upstream, or waiting for another PR to be merged first. 

A good example is when we add functionality to work around a known upstream bug (https://github.com/PennyDreadfulMTG/Penny-Dreadful-Tools/pull/4159), with the intention to remove the workaround when it is fixed upstream (https://github.com/PennyDreadfulMTG/Penny-Dreadful-Tools/pull/4632)

It would be nice to create that "revert PR" at the same time as the original, and use a command like `bors r++` to tell bors "We trust this PR, attempt to stage it every time master is updated".

**Comment by JensRantil** [comment]
🤔 Could the title of this issue better reflect the description?

**Comment by silasary** [comment]
What would you recommend?

**Comment by JensRantil** [comment]
How about "Support for unapproved PR reverts"? I'm having a hard time how the description ties into retries.
